install_pim/docker/installation_docker.rst

```diff
@@ -29,15 +29,18 @@ Configuring you package manager
 
 The default values defined in `docker-compose.yml` match the default values for the `composer` and `yarn` directories:
  - Composer: `~/.composer`
+ - Composer cache directory: `~/.cache/composer`
  - Yarn: `~/.cache/yarn`
 
+On some older versions of Composer, the default cache director can be located at `~/.composer/cache`
```

Comment:
In fact, both are related. Either both the composer config and cache are in `~/.composer` and `~/.composer/cache`, respectively, or they are in `~/.config/composer` and `~/.cache/composer`.
This difference is, afaik, not linked directly to the version of composer, but to the usage or not of the [XDG specifications](https://specifications.freedesktop.org/) (which may not be installed in Docker images), more precisely the [base directory specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html).

Reply:
Yes, but not only...
For instance, if you look [here](https://github.com/composer/composer/blob/2.2/src/Composer/Factory.php#L134), you will see that Composer will use `~/.composer/cache` even if you're on a XDG compliant architecture.

Reply:
Oh, indeed, interesting.